src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,35 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+
+import usersFromServer from '../../api/users.json';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+const getUserById = userId =>
```

Comment:
Maybe it is better to write this logic in the main file.It will make the code more understandable

Reply:
> Maybe it is better to write this logic in the main file.It will make the code more understandable

Yes, you should prepare data and pass it to the components ready to use data so they just render UI.